cmd/snap/cmd_snap_op.go

```diff
@@ -416,6 +416,7 @@ func (x *cmdInstall) installOne(name string, opts *client.SnapOptions) error {
 	}
 	if e, ok := err.(*client.Error); ok && e.Kind == client.ErrorKindSnapAlreadyInstalled {
 		fmt.Fprintf(Stderr, e.Message+"\n")
+		fmt.Fprintf(Stderr, i18n.G("Do you mean 'snap refresh'? See 'snap refresh --help'\n"))
```

Comment:
As discussed, I thought about just running refresh directly, but there are some edge cases which seem problematic. For example, "snap install foo" always means from stable, while "snap refresh foo" means hold what you have. It'd also be curious to have the "refresh" command in the first place if install does the same.

So, I suggest just a mild change to the messaging above. Instead of having a separate line, let's just do something like ` (see "snap refresh --help")` appended to the end of the message. This is using the exact same format we already have in main.go for the login command. This follows your suggestion, and solves the main problem stated and we don't risk opening another can of worms.